src/client/app/heroes/heroes/heroes.component.html

```diff
@@ -3,7 +3,7 @@
     <button mat-raised-button color="primary" type="button" (click)="getHeroes()" matTooltip="Refresh the heroes">Refresh</button>
     <button mat-raised-button color="primary" type="button" (click)="enableAddMode()" *ngIf="!addingHero && !selectedHero" matTooltip="Add a new hero">Add</button>
   </div>
-  <app-filter entityType="Hero" filterPlaceholder="Filter by each hero's name"
+  <app-filter [entityService]="heroService" filterPlaceholder="Filter by each hero's name"
```

Comment:
I always felt that passing a public service into a binding was bad form. I have no evidence of it, but always felt that the child could go get it, too. 

If we do that here, the child gains more code and knowledge (and is not longer container/presenter).

No action needed right now, but let's talk about it.